src/core/client/admin/components/BanModal.css

```diff
@@ -15,3 +15,34 @@ $ban-modal-text: var(--palette-text-500);
 .sitesToggle {
   margin-bottom: var(--spacing-3);
 }
+
+.domainBanHeader {
+  margin-bottom: var(--spacing-2);
+
+}
+
+
+.form {
+  gap: var(--spacing-3);
+  margin-bottom: var(--spacing-3);
+}
+
+.banFromHeader {
+  margin-bottom: var(--spacing-2);
+  margin-top: 0;
+  font-family: var(--font-family-primary);
+}
+
+.sitesOptions {
+  margin-bottom: var(--spacing-1);
+  gap: var(--spacing-2);
+}
+
+.banDomainOption {
+  margin: var(--spacing-1) 0;
+}
+
+.customizeMessage {
+  align-self: flex-start;
+  color: -var(--palette-grey-800);
```

Comment:
Looks like this has an extra dash in front of it making it invalid? It also looks like this darker grey is getting overridden by the button color as is...can you look into this?

Reply:
ah, you're right. The design shows it being darker. I think I need some input on what to do when the design doesn't fit our component library

Reply:
✅